src/acquisition/fluview/fluview_update.py

```diff
@@ -132,23 +179,201 @@ def get_ilinet_data(row):
     'n_patients': optional_int(row[14]),
   }
 
+def get_clinical_data(row):
+  if row[0] == 'REGION TYPE' and row != [
+    'REGION TYPE', 
+    'REGION', 
+    'YEAR', 
+    'WEEK', 
+    'TOTAL SPECIMENS', 
+    'TOTAL A',
+    'TOTAL B', 
+    'PERCENT POSITIVE', 
+    'PERCENT A', 
+    'PERCENT B'
+  ]:
+    raise Exception('header row has changed for clinical lab data.')
+  if len(row) == 1 or row[0] == 'REGION TYPE':
+    # this is a header row
+    return None
+  if row[4] == 'X':
+    # data is not reported, ignore this row
+    return None
+  # ignore percentage calculations for now
```

Comment:
Just curious why percent positive is ignored for now? Are you planning to add it later?

Reply:
I added it in a commit just now. I was thinking about how to handle missing values - I just let be NULL now.

Reply:
That's great! This will be helpful. In case you didn't know, there's a measure called "ILI+" that's supposed to be a more precise measure flu prevalence than ILI. It's defined as the product of ILI and percent-positive. Since we already have ILI, the addition of percent-positive will allow us to also compute ILI+.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3873365/